pages/repo.tsx

```diff
@@ -1,20 +1,20 @@
-import MainAppBar from "../views/MainAppBar";
-import Footer from "../components/Footer";
-import conn from '../utils/db';
 import { GetServerSideProps, NextPage } from "next";
+import type { Session } from "next-auth/core/types";
+import { getServerSession } from "next-auth/next";
 import React from "react";
-import { renderObjAsTable } from "../utils/data";
+import Footer from "../components/Footer";
+import RudderContext from "../components/RudderContext";
 import { ContributorVector } from "../types/contributor";
-import Contributors2DView, { getContri2DProps } from "../views/Dashboard/contri_2d";
+import type { DbRepoSerializable } from "../types/repository";
+import { getAuthUserId, getAuthUserName } from "../utils/auth";
+import { renderObjAsTable } from "../utils/data";
+import conn from '../utils/db';
+import { getAndSetAnonymousIdFromLocalStorage } from "../utils/rudderstack_initialize";
 import CommitsPerFile from "../views/Dashboard/commitsPerFile";
+import Contributors2DView, { getContri2DProps } from "../views/Dashboard/contri_2d";
+import MainAppBar from "../views/MainAppBar";
 import RepoList, { getRepoList } from "../views/RepoList";
```

Comment:
Apparently, VSCode has this keyboard shortcut to "Organize imports", which removes unused imports and sorts the imports as well. (Shift+Alt+O). I just ran that.